apps/server/src/modules/dispositif/dispositif.service.ts

```diff
@@ -330,7 +341,16 @@ export const saveAndOverwriteDraft = async (
 
   const updatedDispositif = await updateDispositifInDB(id, { ...dispositifToSave, hasDraftVersion: false });
   if (draftDispositif) await deleteDraftDispositif(id);
-  return { updatedDispositif, hasDraftVersion: !!draftDispositif };
+
+  let hasDraftVersion: DispositifStatus.DRAFT | DispositifStatus.UPDATE_TO_VALIDATE | null = null;
```

Comment:
Habituellement une variable commençant par "has" est une booléenne.  Peut-être ce serait plus clair:

```suggestion
  let draftVersionStatus: DispositifStatus.DRAFT | DispositifStatus.UPDATE_TO_VALIDATE | null = null;
```

Reply:
en effet, c'est plus clair, je change !